server/src/sas/CompletionProvider.ts

```diff
@@ -265,26 +280,39 @@ export class CompletionProvider {
     }
   }
 
-  getCompleteItems(position: Position): Promise<CompletionItem[] | undefined> {
+  getCompleteItems(
+    position: Position,
+  ): Promise<CompletionItem[] | CompletionList | undefined> {
     return new Promise((resolve) => {
       this._getZone(position);
+      const prefix = this._getPrefix(position);
```

Comment:
nit: this.popupContext.prefix is already there, no need to calculate again

Reply:
Done